docs/source/models/vlm.rst

```diff
@@ -141,5 +137,4 @@ A full code example can be found in `examples/openai_vision_api_client.py <https
         export VLLM_IMAGE_FETCH_TIMEOUT=<timeout>
 
 .. note::
-    The prompt formatting with the image token ``<image>`` is not needed when serving VLMs with the API server since the prompt will be 
-    processed automatically by the server.
+    There is no need to format the prompt in the API request when since it will be handled by the server.
```

Comment:
Could you elaborate? I made this change because I no longer mention the use of `<image>` token outside of the LLaVA examples (since it may not be available for certain models).

Reply:
My point is that we're using `llava-hf/llava-1.5-7b-hf` as an example for both offline and online in this page, so calling out the unnecessary usage of image token (in this example, `<image>`) makes it clear what "formatting" is actually not needed.

Reply:
OK, perhaps it'll be more appropriate to put it as a comment inside the code block then.